config.js

```diff
@@ -0,0 +1,29 @@
+var chalk = require('chalk');
+
+function getChalkColors(configInput) {
+  // make the read file understandable to chalk.
+  var chalkColors = {};
+  for (var color in configInput.colors) {
+  	if (configInput.colors.hasOwnProperty(color)) {
+  		var colorFunc = chalk;
+      //build up the chalk function
+      configInput.colors[color].forEach(function(style) {
+  			colorFunc = colorFunc[style];
+  		});
+  		chalkColors[color] = colorFunc;
+  	}
+  }
+  return chalkColors;
+}
+
+var configInput = require("./lmeDefaultConfig.json");
+try {
+	// try to get the config file from root file of the application.
+	var providedConfig = require('app-root-path').require("lmeconfig.json");
+  configInput.colors = Object.assign({}, configInput.colors, providedConfig.colors);

```

Comment:
Do we really need
```js
Object.assign({}, configInput.colors, providedConfig.colors)
```
over
```js
Object.assign({}, configInput, providedConfig)
```
Do we really have to specify `color` property of `providedConfig` and `configInput`?
The reason I mentioned this is because in future, if we have more configurations other than `color`, the second method won't make a mess.